### ISSUE TITLE
Is Debug version just unminified or something else in there?

### ISSUE THREAD

**Issue by sytelus** [bowerjs/knockout#10]
I've minification of all js done separately so I'd like to reference unminified filed. But I'm confused here if .debug.js is just unminified or has some more things in it. It would be nice to maintain common naming such as .min.js for minified version and just .js for unminified version and .debug.js for code with debugging constructs such as logging.

**Comment by kohenkatz** [comment]
The only difference between these files, as far as I know, is the minification.  You can see the packaging script in the original Knockout repository.

**Comment by tannerhodges** [comment]
Yes. The difference is that `DEBUG` gets set to `true` instead of `false`. This is handled by [the `buildDebug` function in the Gruntfile](https://github.com/knockout/knockout/blob/e2e6aee8556f852f1473ca4d45b11ab54415ca31/Gruntfile.js#L95-L103).

![`source.push('var DEBUG=true;\n');`](https://cloud.githubusercontent.com/assets/1619485/7946149/07327924-0943-11e5-8dea-190f9c08bd5f.png)

I haven't looked into the specifics, but from the [KnockoutJS Downloads page](http://knockoutjs.com/downloads/):

> [The debug build] exposes additional unsupported private APIs.

So yeah, I use the debug version and just set `DEBUG = false`.
